query-input/src/main/java/de/brandwatch/minianalytics/queryinput/controller/Controller.java

```diff
@@ -0,0 +1,21 @@
+package de.brandwatch.minianalytics.queryinput.controller;
+
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@org.springframework.stereotype.Controller
```

Comment:
You can import the annotation class and just keep `@Controller`, it is much more comon